stories/alien/level2/slides.html

```diff
@@ -83,7 +83,7 @@
 - It may seem shocking. 
 - But when comparing a human genome to a banana genome, there are so many genes that are similar within the two organisms. 
 - Although, from the outside they appear to be very different organisms. 
-- Thinking about what humans do for living and what a banana does, there's a lot of things they do in the same way, like consuming oxygen. 
+- Thinking about what humans do for living and what a banana does, there's a lot of things they do in the same way. 
```

Comment:
Thinking about a human and a banana. There's a lot of things they do in the same way, like consuming oxygen.